compiler/scenario-service/protos/scenario_service.proto

```diff
@@ -183,15 +183,17 @@ message ScenarioError {
 
   message ContractNotVisible {
     ContractRef contract_ref = 1;
-    Party committer = 2;
```

Comment:
I realize this is protobuf done wrong but there is zero reason to aim for backwards compatibility here.